OneCommunityGlobal/HighestGoodNetworkApp
# Alan click dot persons name

This PR fixes the problem where "the “Filter Entries by Project” box is still sometimes loading the data for the viewer rather than the person being viewed. Sometimes it will fix itself and show the right list, but not always. I also made the PopUpBar be fixed to the top".

How to test:
- View tasks and time in the current account you are logged in.
- Open the dashboard of another user by clicking on the dot next to their name.
- Check if the right tasks and time entries are displaying.
- Check if the popup bar is fixed to the top even if we scroll.
- Do the steps above in multiple accounts.
